Lib/test/test_interpreters/test_api.py

```diff
@@ -647,6 +648,68 @@ def test_created_with_capi(self):
                     self.interp_exists(interpid))
 
 
+    def test_remaining_threads(self):
+        r_interp, w_interp = self.pipe()
+
+        FINISHED = b'F'
+
+        # It's unlikely, but technically speaking, it's possible
+        # that the thread could've finished before interp.close() is
+        # reached, so this test might not properly exercise the case.
+        # However, it's quite unlikely and I'm too lazy to deal with it.
```

Comment:
There's no need to immortalize a negative remark about yourself. 😄

```suggestion
        # However, it's quite unlikely and probably not worth bothering about.
```